### DISCUSSION TITLE
Is it possible get get webserver user credentials from k8s Secret?

### DISCUSSION THREAD

**Discussion by okulbida** [apache/airflow#30433] [Q&A]
Hi, 
Is it possible get get webserver user credentials from k8s Secret?
Didn't find such details. I see only https://github.com/airflow-helm/charts/blob/main/charts/airflow/docs/faq/security/set-webserver-secret-key.md which seems a different key

**Comment by potiuk** [comment]
Not by default. Airfow user's credentials are kept in a database and are maintained there. by default. You can integrate it with external authentication, so possibly in theory you coudl have an authentucation based on secret but that would require integrating it with FAB (Flask Application Builder) because it's FAB that provides authentication for web users for airflow https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/security/webserver.html#web-authentication . You can read the docs and ask in FAB if this is an option.
